puree/src/main/java/com/cookpad/puree/PureeLogger.java

```diff
@@ -60,6 +63,13 @@ public void discardBufferedLogs() {
         storage.clear();
     }
 
+    public void discardOldBufferedLogs() {
```

Comment:
I don't like this because it loads all the records in memory, which is absolutely ineffective.

Add `PureeStorage#truncateBufferedLogs(int maxRecords)` and delete records without loading them.